tests/contracts/main/application/test_worker.py

```diff
@@ -275,8 +275,9 @@ def test_bond_worker(testerchain, threshold_staking, pre_application, token_econ
     assert pre_application.functions.getWorkerFromOperator(worker1).call() == NULL_ADDRESS
     assert pre_application.functions.operatorFromWorker(worker1).call() == NULL_ADDRESS
 
-    # Operator can't bond the first worker again because worker is an operator now
     testerchain.time_travel(seconds=min_worker_seconds)
+
+    # Operator can't bond the first worker again because worker is an operator now
```

Comment:
I put it before time travel to show that exception happens even after time before can change worker has passed